monai/networks/nets/resnet.py

```diff
@@ -154,6 +157,71 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return out
 
 
+class Daf3dResNetBottleneck(ResNetBottleneck):
```

Comment:
It might be a better idea to put everything `Daf3d` related into `daf3d.py` rather than added here despite this being a `ResNet` subclass. It's easier to be able to point to one file called `daf3d.py` for all the code from your paper.

Reply:
OK we will do it. Thanks!